modules/core/src/__tests__/__mocks__/mock-config.ts

```diff
@@ -21,6 +21,7 @@ export const configMock = {
     features: {
         applications: {
             scopes: {
+                feature: [ "console:applications" ],
```

Comment:
We have added default values in framework and j2 mappings for these new properties right?

Reply:
Added with this PR - https://github.com/wso2/carbon-identity-framework/pull/3540